model/app/installer_test.go

```diff
@@ -71,8 +74,8 @@ func manifestKonnector() string {
 }`
 }
 
-func serveGitRep(dir string) context.CancelFunc {
-	localGitDir = dir
+func serveGitRep(t *testing.T) (context.CancelFunc, string) {
```

Comment:
[nit] I would have swapped the returned variables: string first, context.CancelFunc second.